.travis.yml

```diff
@@ -45,6 +62,11 @@ install:
     - travis_retry pip install tox
     - python -VV
     - curl-config --version; pip freeze
+    - |
+      if [[ -n ${BUILD_WHEEL} ]]; then
+         pip install -U cibuildwheel
+         cibuildwheel --output-dir dist/
```

Comment:
`cibuildwheel` is relatively slow; we shouldn't run it until the end of `script` instead of here at the end of `install` (Alternatively, we could use `cibuildwheel` as the first step in all cases, and have tox test the built wheel instead of building its own). 

It might be better to have each travis configuration build its own wheel instead of building them all from py38 (to ensure we test what we build, and also to have some parallelism in the CI pipeline). But I'd also like to move to the stable ABI so we'd no longer have per-python-version wheels.

Reply:
Added cibuildwheel as first step in each case (as suggested). also installing tornado from it's own build wheel and running test on the same.